### PR TITLE
feat: Add Multiple Event "Documents" to the system

### PR DESCRIPTION
**Pull request by ranjitkshah**
reqd for [#7335](https://github.com/fossasia/open-event-frontend/issues/7335)

#### Short description of what this resolves:
 added document_enabled field in event model

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by ranjitkshah** [comment]
need to discuss in meeting

**Comment by sachinchauhan2889** [comment]
We get this error when we try to fetch eventsList after login. 

We did not get error when we are logout and when we fetch a particular event

**Comment by sachinchauhan2889** [comment]
cause of this Error.




api of event. 
line no 166

**Comment by ranjitkshah** [comment]
`curl 'http://localhost:5000/v1/users/3/events?include=general-statistics%2Croles.role%2Croles.user%2Ctickets&filter=%5B%7B%22name%22%3A%22state%22%2C%22op%22%3A%22eq%22%2C%22val%22%3A%22published%22%7D%2C%7B%22or%22%3A%5B%7B%22name%22%3A%22starts-at%22%2C%22op%22%3A%22ge%22%2C%22val%22%3A%222021-06-10T15%3A07%3A42.033Z%22%7D%2C%7B%22and%22%3A%5B%7B%22name%22%3A%22starts-at%22%2C%22op%22%3A%22le%22%2C%22val%22%3A%222021-06-10T15%3A07%3A42.033Z%22%7D%2C%7B%22name%22%3A%22ends-at%22%2C%22op%22%3A%22gt%22%2C%22val%22%3A%222021-06-10T15%3A07%3A42.034Z%22%7D%5D%7D%5D%7D%5D&page%5Bsize%5D=10&page%5Bnumber%5D=1&sort=starts-at' \
  -H 'Connection: keep-alive' \
  -H 'sec-ch-ua: " Not A;Brand";v="99", "Chromium";v="90", "Google Chrome";v="90"' \
  -H 'Accept: application/vnd.api+json' \
  -H 'Authorization: JWT eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MjMzMzU1NzMsIm5iZiI6MTYyMzMzNTU3MywianRpIjoiNWRkMmU5ZGItY2IxMS00YWFjLTkzNDItNmM0MzMwZDBkZTQ3IiwiZXhwIjoxNjIzNDIxOTczLCJpZGVudGl0eSI6MywiZnJlc2giOnRydWUsInR5cGUiOiJhY2Nlc3MiLCJjc3JmIjoiY2Q1MDRiZTctODQ5Yy00ZDkyLWFkNjEtYmFiZjE2ZjE3NDdhIn0.8RMiowitXplBsrWzMlGIoz3Ym3O8AVHc5LtKSMyyXe8' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36' \
  -H 'Content-Type: application/vnd.api+json' \
  -H 'Origin: http://localhost:4200' \
  -H 'Sec-Fetch-Site: same-site' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Referer: http://localhost:4200/' \
  -H 'Accept-Language: en-GB,en-US;q=0.9,en;q=0.8' \
  --compressed`

**Comment by ranjitkshah** [comment]
@prateekj117 please check

**Comment by ranjitkshah** [comment]
> @prateekj117 please check

using JSONB() instead of JSONB() in migrations but it is not working

`File "/home/ranjitkshah/opensource/eventyay/open-event-server/migrations/versions/rev-2021-06-08-00:18:31-66a3114b2317_.py", line 21, in upgrade
    op.add_column('events', sa.Column('document_links', sa.JSONB(), nullable=True))
AttributeError: module 'sqlalchemy' has no attribute 'JSONB'`

**Comment by ranjitkshah** [comment]
> @prateekj117 please check

using JSONB() instead of JSON() in migrations but it is not working

`File "/home/ranjitkshah/opensource/eventyay/open-event-server/migrations/versions/rev-2021-06-08-00:18:31-66a3114b2317_.py", line 21, in upgrade
    op.add_column('events', sa.Column('document_links', sa.JSONB(), nullable=True))
AttributeError: module 'sqlalchemy' has no attribute 'JSONB'`

File: app/models/event.py [hunk 1]
```diff
@@ -62,6 +62,8 @@ class Privacy:
     is_promoted = db.Column(db.Boolean, default=False, nullable=False)
     is_demoted = db.Column(db.Boolean, default=False, nullable=False)
     is_chat_enabled = db.Column(db.Boolean, default=False, nullable=False)
+    is_document_enabled = db.Column(db.Boolean, default=False, nullable=False)
+    document_links = db.Column(db.JSON)
```
Comment (prateekj117) [inline app/models/event.py:66]: Import `JSONB` data type `from sqlalchemy.dialects.postgresql import JSONB`. 
Document links column would look like this:
```
document_links = db.Column(JSONB)
```
The schema would probably remain the same.

**Review by prateekj117 — CHANGES_REQUESTED** [review]

File: app/api/schema/events.py [hunk 2]
```diff
@@ -12,6 +13,10 @@
 from app.models.event import Event
 
 
+class DocumentLinkSchema(Schema):
+    name = fields.String(required=True)
+    link = fields.String(required=True)
+
```
Comment (prateekj117) [inline app/api/schema/events.py:19]: Add one more new line here.

Ref: https://www.python.org/dev/peps/pep-0008/#blank-lines

**Review by prateekj117 — CHANGES_REQUESTED** [review]

**Review by MeghalBisht — APPROVED** [review]

**Review by prateekj117 — APPROVED** [review]
LGTM !!!

**Pull request merged**